.github/workflows/ci.yml

```diff
@@ -106,3 +106,18 @@ jobs:
           HUSKY: 0
           NPM_TOKEN: ${{secrets.NPM_TOKEN}}
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+  chromatic-deployment:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3.1.0
+        with:
+          fetch-depth: 0
+      - uses: pnpm/action-setup@v2.2.3
+      - name: Install dependencies
+        run: pnpm install
+      - name: Publish to Chromatic
+        if: github.ref != 'refs/heads/main'
+        uses: chromaui/action@v1
```

Comment:
Can we retrieve storybookUrl outputted by this action to deploy a new GitHub environment (thus removing our deploy) ? https://github.com/chromaui/action/blob/b9b365f0c7d27357f2f73211bd3c9a58691d2aaf/action.yml#L119

Reply:
@chambo-e should be better 😄